index.html

```diff
@@ -4,16 +4,92 @@
+      </nav>
+    </header>
+    <section class="top-section">
+      <img src="img/profile.jpg" alt="This Cat - Hire!" class="profile-img">
+      <article>
+        <h1>This Cat - Working Cat</h1>
+        <h4>Warming laps and spreading fur since 2003!</h4>
+        <p>
+        This Cat doesn't know English (yet) but this website is  curated by individuals who can book her for on-site or  remote work.
+        </p>
+        <button class="button-hire">Hire This Cat!</button>
+      </article>
+    </section>
+    <section class="middle-section">
+      <div class="middle-section-column">
+        <article>
+          <h2>This Cat's Abridged Manifesto</h2>
+          <p>
+              Claw drapes favor packaging over toy yet meow loudly just to annoy owners. My left donut is missing, as is my right. Catasstrophe meow to be let out so nya nya nyan a nice warm laptop for me to sit on but bite off human's toes purr while eating. Small kitty warm kitty little balls of fur a nice warm laptop for me to sit on, and floof tum, tickle bum, jellybean footies curly toes pet my belly, you know you want to; seize the hand and shred it! scamper yet climb a tree, wait for a fireman jump to fireman then scratch his face. If it fits, i sits purr, and sit on human they not getting up ever. Inspect anything brought into the house kitty power or love you, then bite you with tail in the air hit you unexpectedly chase ball of string or meowzer. Weigh eight pounds but take up a full-size bed meow for it's 3am, time to create some chaos catch mouse and gave it as a present for and sometimes switches in french and say "miaou" just because well why not and twitch tail in permanent irritation and lies down.
+          </p>
+        </article>
+      </div>
+      <div class="middle-section-column">
+        <article class="skills">
+          <h3>Skills:</h3>
+          <ul>
+            <li>Sleeping</li>
+            <li>Eating</li>
+            <li>Lap warming</li>
+            <li>Monitoring outdoor activities</li>
+            <li>Depositing fur on soft surfaces</li>
+          </ul>
+        </article>
+      </div>
+    </section>
+    <section class="bottom-section">
+      <article class="work-history">
+        <h3>Work History</h3>
+        <table>
+          <tr>
+            <td>Yrs</td>
+            <td>Company</td>
+            <td>Title</td>
+            <td>Tech Used</td>
+          </tr>
+          <tr>
+            <td>3</td>
+            <td>Apple</td>
+            <td>Cat</td>
+            <td>Nap</td>
+          </tr>
+          <tr>
+            <td>2</td>
+            <td>Microsoft</td>
+            <td>Cat</td>
+            <td>Eat</td>
+          </tr>
+          <tr>
+            <td>3</td>
+            <td>Amazon</td>
+            <td>Cat</td>
+            <td>Monitor</td>
+          </tr>
+        </table>
+      </article>
+    </section>
+    <footer>
+      Copyright This Cat 2018
+    </footer>
+  </div>
```

Comment:
I had little tears of joy when I saw your beautifully laid out HTML. Tell cat, good job!